Week-2/Html Advanced/lineUp.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Select Line Up</title>
+</head>
+<body>
+    <form>
+        <p>
+            <fieldset>
+                <legend>Status:</legend>
+                <label >
+                    <input type="radio" name="status" value="Draft">
+                    Draft
+                </label>
+                <label>
+                    <input type="radio" name="status" value="Final" checked>
+                    Final
+                </label>
+            </fieldset>
+        </p>
+
+        <p>
```

Comment:
In HTML, it's generally not considered good practice to place a <label> and a <select> element inside a <p> (paragraph) tag. The <p> tag is typically used for defining paragraphs of text, and it's semantically intended for content, not form elements. While browsers will likely render it without issues, it's a better practice to use appropriate HTML elements to structure your content and forms.

To improve the code's structure and adhere to best practices, you can use a <form> element, and within it, you can place the <label> and <select> elements. Here's an example of how you can structure your code:

```
<form>
    <label for="gameDate">Date:</label>
    <select name="gameDate" id="gameDate">
        <option value="March 20">March 20</option>
        <option value="March 24">March 24</option>
        <option value="April 2">April 2</option>
        <option value="April 6">April 6</option>
        <option value="April 13">April 13</option>
    </select>
</form>
```